vector/src/main/java/fr/gouv/tchap/activity/TchapRoomCreationActivity.java

```diff
@@ -66,6 +69,8 @@
 import im.vector.activity.VectorRoomInviteMembersActivity;
 import im.vector.util.VectorUtils;
 
+import static im.vector.activity.CommonActivityUtils.REQUEST_CODE_PERMISSION_ROOM_DETAILS;
```

Comment:
Remove this import, and check the other added imports (I think they are not relevant)